conode/Dockerfile-dev

```diff
@@ -9,4 +9,4 @@ RUN ln -s /conode_data .config/conode
 
 EXPOSE 7770 7771
 
-CMD "./run_nodes.sh -d /conode_data -v 2"
+CMD ["/root/run_nodes.sh", "-d", "/conode_data", "-v", "2"]
```

Comment:
By curiosity,is there any reason why in cothority it's using relative path and not here?

Reply:
Oups, old code, I'm pushing a relative path fix.

Reply:
Haha, yea, now that I'm looking again at https://github.com/dedis/cothority/pull/1976, I see that one is full, the other, relative; in order to follow @jeffallen's directions, I'm copying as it is.

Reply:
The docker files from cothority does not exactly do the same as the ones in template:

||Dockerfile|Dockerfile-dev|
|-|-|-|
|Cothority|conode -d 2 server|run_nodes.sh -n 1|
|Template|run_nodes.sh -d /conode_data -v 2|run_nodes.sh -d /conode_data -v 2|

`run_nodes.sh` creates by default 3 conodes. Therefore, by syncing from Cothority, it now always creates one conode.

Reply:
Okay, so I'm keeping the old behavior with https://github.com/dedis/cothority_template/pull/54/commits/83c0b8e59c33e95725c662bea62a33d5b82ba223.

Reply:
Also check that commands from the Makefile still work. Apparently there need to be some updates there too for the build to work. 🌈

Reply:
Yep, so I copied it from cothority (and removed the `tooldist` target as irrelevant here) in https://github.com/dedis/cothority_template/pull/54/commits/a8680e4812b7c339930924ae46b1e02ff100efaa